tests/api.c

```diff
@@ -18570,6 +18696,66 @@ static int test_wc_RNG_GenerateBlock()
 }
 #endif
 
+static void test_wolfSSL_X509_CRL(void)
+{
+#if defined(OPENSSL_EXTRA) && defined(HAVE_CRL)
+
+    X509_CRL *crl;
+    char pem[][100] = {
+        "./certs/crl/crl.pem",
+        "./certs/crl/crl2.pem",
+        "./certs/crl/caEccCrl.pem",
+        "./certs/crl/eccCliCRL.pem",
+        "./certs/crl/eccSrvCRL.pem",
+        ""
+    };
+
+#ifdef HAVE_TEST_d2i_X509_CRL_fp
```

Comment:
This option is to skip Jenkins test because current sample certs miss CRL in DER format.
I assume we should update crlgen.sh and create some files at certain moment. For now, the der files were manually generated and by enabling HAVE_TEST_d2i_X509_CRL_fp at local environment.